infra/config_rpc_disabled.toml

```diff
@@ -1,6 +1,7 @@
 otlp_collector_endpoint = "http://otel-collector:4317"
 
 [[nodes]]
+enabled_apis = [ "admin", "erigon", "eth", "net", "ots", "trace", "txpool", "web3", "zilliqa" ]
```

Comment:
This config should be empty! As the equivalent of `disable_rpc`

Reply:
I'm dumb, fixed - Thanks!